pom.xml

```diff
@@ -204,7 +204,7 @@
                             <execution>
                                 <id>run-copyright</id>
                                 <goals>
-                                    <goal>check</goal>
+                                    <goal>copyright</goal>
```

Comment:
Copyright was failing because of the new classes I added, but in the error it was not written which files are failing this check.

@Verdent asked me to write this in my commit, so we don't need to create a new pull request. With this change we will see the files that are not successful with the copyright.